File: fizzbuzz.py
import sys

passArg = False if len(sys.argv) < 2 is "" else True

n = int(sys.argv[1]) if passArg else 15  


for i in range(0, n):
  s = "%d : " % i
  s += "fizz" if i % 3 == 0 else ""
  s += "buzz" if i % 5 == 0 else ""
  print(s)

File: fizzbuzz.c
#include <stdio.h>
#include <string.h>



int main(int argc, char *argv[]){
  int i;
  int n = atoi(*(argv+1));
  printf("%d\n", n);
  for(i = 0; i < n; i++){
    printf("%d : ", i);
   if( i % 3 == 0) {
      printf("fizz");
     }
    if (i % 5 == 0) {
      printf("buzz");
    }
    printf("\n");
  }

  return 0;
}
